todo/src/App.js

```diff
@@ -0,0 +1,63 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import {
+  addTodo,
+  toggleComplete,
+  removeCompleted,
+  deleteTodo
+} from "./actions";
+import Form from "./components/Form";
+import TodoList from "./components/TodoList";
+
+import "./css/index.css";
+
+class App extends Component {
+  toggleComplete = id => {
+    this.props.toggleComplete(id);
+  };
+
+  handleDelete = (ev, id) => {
+    ev.preventDefault();
+    ev.stopPropagation();
+
+    this.props.deleteTodo(id);
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <h1>Todo List:</h1>
```

Comment:
instead you can write 
```
state = {
 newTodo: ''
}
```
it is exactly the same thing, take a look at babel transpiles

[The constructor is dead, long live the constructor!](https://hackernoon.com/the-constructor-is-dead-long-live-the-constructor-c10871bea599)